runtime/src/snapshot_package.rs

```diff
@@ -129,6 +129,19 @@ pub struct SnapshotPackage {
     pub snapshot_type: SnapshotType,
 }
 
+impl SnapshotPackage {
+    pub fn get_snapshot_type(&self) -> Option<SnapshotType> {
+        let file_name = self.path().file_name().unwrap().to_str().unwrap();
+        if file_name.starts_with("snapshot") {
+            Some(SnapshotType::FullSnapshot)
+        } else if file_name.starts_with("incremental-snapshot") {
+            Some(SnapshotType::IncrementalSnapshot(0))
+        } else {
+            None
+        }
+    }
+}
```

Comment:
`SnapshotType` is in `SnapshotPackage`, so you can already get the snapshot type with `snapshot_package.snapshot_type`.

https://github.com/solana-labs/solana/blob/09b58e1cfbb7b568e8b4b677a889da96163ea38f/runtime/src/snapshot_package.rs#L129

Reply:
Fixed